### DISCUSSION TITLE
Theme and font from a config

### DISCUSSION THREAD

**Discussion by nnra6864** [FreeTubeApp/FreeTube#6285] [Q&A] [answered]
Hi, I wrote a tool, [Ricer](https://github.com/nnra6864/Ricer), that I'm using to rice my entire system.
I was wondering if there are any plans to add config support, such as for theme and font, so that I could use my tool with your platform in order for it to match my system colors and font.
Thanks in advance!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment, marked as answer]
@nnra6864 thankyou for you interest in our project. It is a very neat tool i must say but i have to inform you that we dont have plans to integrate this. Our user base consists of many novice type of users and very little power users. Integrating this for a very small subset of our users just doesnt make sense.

**Comment by nnra6864** [comment]
I see, that makes perfect sense to me.
Thanks for the quick response!
